README.md

```diff
@@ -1 +1,72 @@
 # babel-plugin-transform-react-pug
+
+The official means by which you can use pug in your react components, replaces the use of react-jade when moving to pug.
+
+This plugin transforms the pug inside of your react components.
+
+## Installation
+```
+npm install babel-plugin-transform-react-pug
+```
+## Usage
+.babelrc
+```js
+{
+  "presets": ["forbeslindesay"],
+  "plugins": [
+    "transform-react-pug",
```

Comment:
we should not be forcing people to use the "forbeslindesay" preset, instead they should add "transform-react-jsx".